websites/views.py

```diff
@@ -167,6 +171,38 @@ def get_serializer_class(self):
         else:
             return WebsiteStarterDetailSerializer
 
+    @action(detail=False, methods=["post"], permission_classes=[])
+    def site_configs(self, request):
+        """Process webhook requests for WebsiteStarter site configs"""
+        data = json.loads(request.body)
+        if data.get("repository"):
+            if not valid_key(settings.GITHUB_WEBHOOK_KEY, request):
+                return Response(status=status.HTTP_403_FORBIDDEN)
+            files = [
+                file
+                for sublist in [
+                    commit["modified"] + commit["added"] for commit in data["commits"]
+                ]
+                for file in sublist
+                if file.endswith(settings.OCW_STUDIO_SITE_CONFIG_FILE)
```

Comment:
Can we make this a more specific comparison, perhaps `if os.path.basename(file) == settings.OCW_STUDIO_SITE_CONFIG_FILE`?